plugins/Admin/admin.js

```diff
@@ -48,4 +49,21 @@ exports.uptime = {
 		}
 		msg.channel.send("**Uptime**: " + timestr);
 	}
-}
\ No newline at end of file
+}
+
+exports.userRole = {
+    usage: "<@user> <@role>",
+    description: "Used to toggle @role of specified @user",
+    process: function(bot,msg,arg){
+        var args = arg.split(" ");
+        var user, role;
+        if(typeof args[0] != 'undefined' && typeof args[1] != 'undefined'){
```

Comment:
hmm at this point we have enough code that we might benefit from adding a linter/formatter. should always use === or !== ;)

Reply:
never really used a linter before so it will take me some time to get used to it and change my old habits. will quickly fix those as well.